src/olympia/addons/views.py

```diff
@@ -141,14 +140,6 @@ def extension_detail(request, addon):
         return render(request, 'addons/impala/details.html', ctx)
 
 
-def _category_personas(qs, limit):
-    def fetch_personas():
-        return randslice(qs, limit=limit)
-    # TODO: .query_key comes from cache-machine, find replacement
-    key = make_key('cat-personas:' + qs.query_key(), normalize=True)
-    return cache_get_or_set(key, fetch_personas)
```

Comment:
Are you sure we can remove this caching? The query is nasty.

Reply:
I may optimize this a bit, I'm testing this currently a bit more in-depth. I'm testing the whole view though so the solution might be a bit more than just caching this query.

Reply:
tldr; this is the first patch removing cache-machine, more performance optimizations may be added in later PRs. That part on it's own doesn't add too much more stress on the database from what I found, can be optimized though (the whole view can)

Reply:
I added it back as suggested in 72664b17aa